chain/indexer/src/config.rs

```diff
@@ -18,6 +20,27 @@ pub struct IndexerConfig {
     /// Default is `true`.
     #[default(true)]
     pub fast_sync: bool,
+
+    /// Whether to perform logs snapshot download on startup.
+    /// When enabled, it allows for quicker indexing from scratch.
+    ///
+    /// Default is `false`.
+    #[default(false)]
+    pub enable_logs_snapshot: bool,
+
+    /// URL to download logs snapshot from.
+    /// This should point to a publicly accessible tar.gz file containing
+    /// the SQLite logs database files.
+    ///
+    /// Default is None
+    #[default(None)]
+    pub logs_snapshot_url: Option<String>,
+
+    /// Path to the data directory where databases are stored.
+    /// This is used for snapshot installation and database state checking.
+    ///
+    /// Default is empty string (must be set by application).
+    pub data_directory: String,
```

Comment:
is this not part of the top level db_dir? Could that value not be propagated somehow? what if those 2 values differ?

Reply:
This is just passed down by the configuration and equals to `cfg.db.data`. This was the best way to propagate that value into the indexer.